packages/map/lib/src/leaflet/classes/leaflet-map.ts

```diff
@@ -21,56 +19,67 @@ export class LeafletMap {
 	};
 	public mode = this.modes.DRAGGING;
 
-	constructor(public options: LeafletMapOptions) {
+	constructor(public options: LeafletMapOptions,
+				public mapService: MapService) {
 	}
 
 	// LIFECYCLE
 	init(element: any) {
-		this.initialized = true;
-		this.map = L.map(element, {
-			center: this.options.center,
-			zoom: this.options.zoom,
-			attributionControl: false,
-			zoomControl: false,
-			scrollWheelZoom: false,
-		});
-		this.onInit.emit();
+		if (this.mapService.L !== null && this.mapService.esri !== null) {
```

Comment:
I would rather see this check integrated in the mapService, something like `if (this.mapService.isAvailable())`